tests/console/hostname.pm

```diff
@@ -22,6 +22,7 @@ sub run() {
 
     script_run "hostname";
     assert_screen("hostname-$hostname");
+    assert_script_run "systemctl -q is-active network.service && systemctl restart network.service";
```

Comment:
okay, reluctant +1 - dislike it, but understand why it's there and can't think of anything better